docs/DeveloperGuide.md

```diff
@@ -307,6 +307,58 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 3a1. AddressBook shows an error message.
 
       Use case resumes at step 2.
+    
+**Use case: Add leaves to an employee**
+
+**MSS**
+
+1.  User requests to list employees
+2.  HeRon shows a list of employees
+3.  User requests to add a certain number of leaves to a specific employee in the list
+4.  HeRon adds the leaves to the employee
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+
+    * 3a1. AddressBook shows an error message.
+
+      Use case resumes at step 2.
+
+**Use case: Remove leaves from an employee**
+
+**MSS**
+
+1.  User requests to list employees
+2.  HeRon shows a list of employees
+3.  User requests to remove a certain number of leaves from a specific employee in the list
+4.  HeRon removes the leaves from the employee
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is invalid.
+
+    * 3a1. AddressBook shows an error message.
+
+      Use case resumes at step 2.
+
```

Comment:
Similar to the previous comment, could also consider if the inputted leaves is given as a non-numeric character, or if a negative number is given.

Reply:
As mentioned in the preceding conversation, I have added the extension for this use case also.